internal/pkg/controller/controller_test.go

```diff
@@ -0,0 +1,91 @@
+package controller
+
+import (
+	"time"
+	"math/rand"
+
+	"github.com/stakater/Reloader/pkg/kube"
+	"k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	client, _     = kube.GetClient()
```

Comment:
Without checking error is a bad practice. Make sure there's no error or there's no point in going forward to the tests